website/docs/components/breadcrumb/index.md

```diff
@@ -1,5 +1,7 @@
 ---
 title: Breadcrumb
+description: A breadcrumb is a type of secondary navigation that reveals the user's location in a website.
```

Comment:
We use "application" more than we use "website", so probably good to keep them consistent.

```suggestion
description: A breadcrumb is a type of secondary navigation that reveals the user's location in an application.
```

Reply:
I sense a bit of inconsistency with this description, in a sense that we state it as a definition [a Horse is a mammal that...], whereas in other places we don't mention the component name [a mammal that...]